_extras/guide.md

```diff
@@ -3,293 +3,548 @@ layout: page
-    merging files with different line endings.  (Even if everyone's on
-    some flavor of Unix, different editors may or may not add a
-    newline to the last line of a file.) Take a moment to explain
-    these issues, since learners will almost certainly trip over them
-    again.  If learners are running into line ending problems, GitHub
-    has a [page][github-line-endings] that helps with troubleshooting.
-
-*   We don't use a Git GUI in these notes because we haven't found one that
-    installs easily and runs reliably on the three major operating systems, and
-    because we want learners to understand what commands are being run.  That
-    said, instructors should demo a GUI on their desktop at some point during
-    this lesson and point learners at [this page][github-gui].
-
-*   Instructors should show learners graphical diff/merge tools like
-    [DiffMerge][diffmerge].
-
-*   When appropriate, explain that we teach Git rather than CVS, Subversion, or
-    Mercurial primarily because of GitHub's growing popularity: CVS and
-    Subversion are now seen as legacy systems, and Mercurial isn't nearly as
-    widely used in the sciences right now.
-
-*   Further resources:
-    *    [git-it][git-it] is a self-paced command-line Git demo,
-         with [git-it-electron][git-it-electron] its GitHub Desktop successor.
-    *    [Code School][code-school] has a free interactive course, [Try Git][try-git].
-    *    for instructors, [the Git parable][git-parable] is useful background reading
-
 ## [Automated Version Control]({{ page.root }}/01-basics/)
 
-*   Ask, "Who uses 'undo' in their editor?" All say "Me". 'Undo' is the simplest
-    form of version control.
+Use slides
 
-*   Give learners a five-minute overview of what version control does for them
-    before diving into the watch-and-do practicals.  Most of them will have
-    tried to co-author papers by emailing files back and forth, or will have
-    biked into the office only to realize that the USB key with last night's
-    work is still on the kitchen table.  Instructors can also make jokes about
-    directories with names like "final version", "final version revised",
-    "final version with reviewer three's corrections", "really final version",
-    and, "come on this really has to be the last version" to motivate version
-    control as a better way to collaborate and as a better way to back work up.
+### Key Points
+* Version control is like an unlimited "undo".
+* Version control allows many people to work in parallel.
 
 ## [Setting Up Git]({{ page.root }}/02-setup/)
 
-*   We suggest instructors and students use `nano` as the text editor for this
-    lessons because
-
-    *   it runs in all three major operating systems,
-    *   it runs inside the shell (switching windows can be confusing to students), and
-    *   it has shortcut help at the bottom of the window.
+Use the setup page here.
```

Comment:
these were added to the slide deck as well

Reply:
Ok I'll refer to the deck instead then.